app/views/template_invocations/_output_line_set.html.erb

```diff
@@ -2,6 +2,6 @@
   <%= content_tag :div, :class => 'line ' + output_line_set['output_type'], :data => { :timestamp => output_line_set['timestamp'] } do %>
 
     <%= content_tag(:span, (@line_counter += 1).to_s.rjust(4).gsub(' ', '&nbsp;').html_safe + ':', :class => 'counter', :title => (output_line_set['timestamp'] && Time.at(output_line_set['timestamp']))) %>
-    <%= content_tag(:div, (line.empty? ? '&nbsp;' : h(line)).html_safe, :class => 'content') %>
+    <%= content_tag(:div, (line.empty? ? '&nbsp;' : h(colorize_line(line))).html_safe, :class => 'content') %>
```

Comment:
It should be colorize_line(h(line)), otherwise you would see 
![rex-ansi](https://cloud.githubusercontent.com/assets/190443/22510285/afa2588e-e891-11e6-8f3d-af985b2f6bfe.png)